Monika After Story/game/script-windowreacts.rpy

```diff
@@ -586,3 +586,1108 @@ label mas_wrs_twitch:
+                "skip alert": None,
+                "keep_idle_exp": None,
+            },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_presentation:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Working on a presentation, [player]? What's it about?"
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_presentation')
+    return      
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_spreadsheet",
+            category=["- Excel"|"- Google Sheets"|"LibreOffice Calc"],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+            },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_spreadsheet:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Spreadsheets are pretty underrated these days, don't you think?",
+           "What are you working on here, [player]?"
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_spreadsheet')
+    return      
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_access",
+            category=["- Access"],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+            },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_access:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Doing some database work, [player]?",
+           "This stuff looks pretty tough to work with. You're so smart!"
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_access')
+    return      
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_publisher",
+            category=["- Publisher"],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+            },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_publisher:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "What are you designing here, [player]?",
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_publisher')
+    return  
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_onenote",
+            category=["- OneNote"],
```

Comment:
I'm not a user of this software, but from what I found online it uses just `OneNote`. Confirm please.